src/components/rangeInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react"
+
+import Input from "./input/index"
+
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
+  value?: {
+    min: number
+    max: number
+  }
+  isValid?: {
+    min: boolean
+    max: boolean
+  }
+  placeholder?: {
+    min: string
+    max: string
+  }
+  required?: boolean
+  rangeInputLabel?: string
+  initialValue?: string
```

Comment:
What's the difference between `initialValue` and `value`? Why is this one a string?